python/adelphi/bin/adelphi

```diff
@@ -79,9 +128,32 @@ def export_gemini(ctx):
             log.error("Gemini schema doesn't support multiple keyspaces.")
             exit(2)
 
-    my_fn = partial(cluster_fn, my_export_fn, ctx.obj['keyspace_names'], build_options(ctx))
+    my_fn = partial(cluster_fn, my_export_fn, ctx)
     with_cluster(my_fn, **({k:ctx.obj[k] for k in ["hosts","port","username","password"]}))
 
 
+@export.command()
+@click.option('--token', help='Access token for schema storage on Github')
+# TODO: Change the default to something real when we have something real
+@click.option('--repo', default="absurdfarce/schema-playground", help='Name of the repo used for schema storage on Github')
+@click.option('--author', help='Github username to be used for schema commits')
+@click.pass_context
+def github(ctx, token, repo, author):
+
+    if ctx.obj["anonymize"]:
+        log.error("All schemas contributed to Adelphi must be anonymized")
+        exit(6)
```

Comment:
We want to explicitly reject the contribution of non-anonymized schemas to the repo